platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -75,7 +75,7 @@ export default class RetrieveMetadataLoaderAsync extends RetrieveMetadataLoader
 
     const client = new StaticWadoClient({
       ...server,
-      url: server.qidoRoot,
+      url: server.wadoRoot,
```

Comment:
This is tricky because the first step of this loader is to perform a QIDO-RS searchForSeries query

https://github.com/OHIF/Viewers/pull/2712/files#diff-5055a070c8627f67d3e30330840d741fe5b0c4b263fc4a952da801dfc955df41R106

Reply:
Thank you Erik for your review. I’m closing this PR until I find a solution that will take into account your feedback. Will leave the issue open though.